grunt_tasks/plugin.js

```diff
@@ -468,6 +537,26 @@ module.exports = function (grunt) {
         configurePluginForBuilding(path.resolve(grunt.config.get('pluginDir'), name), true);
     });
 
+    // This is noop that exists for backwards compatibility in case any plugins rely
+    // on its existence.
+    grunt.registerTask(
```

Comment:
Let's add `@deprecated: remove in v3` here. I grep -i for "deprecated" to find things to remove :)